apps/tailwind-components/components/input/Boolean.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <InputRadioGroup
+    :id="id"
+    :modelValue="props.modelValue"
+    @update:modelValue="onInput"
+    @focus="$emit('focus')"
+    @blur="$emit('blur')"
+    :radioOptions="radioOptions"
```

Comment:
is this not a bit weird? Did you consider alternatives? E.g. a trueLabel, falseLabel en and if applicable something with nullflavors?